yagta/.ailocalcontext/context.md

```diff
@@ -0,0 +1,205 @@
+# GraphQL Testing API - AI Development Context
```

Comment:
Would all AI code editors pick this up, or should we move this to AGENT.MD instead?